templates/etc/vimrc.j2

```diff
@@ -75,3 +75,6 @@ set suffixes=.bak,~,.swp,.o,.info,.aux,.log,.dvi,.bbl,.blg,.brf,.cb,.ind,.idx,.i
 
 " Enable modelines if there are some in files.
 set modeline
+
+" make a copy of the file and overwrite the original one
+set backupcopy=yes
```

Comment:
Please make `backupcopy` configurable using a variable. The default should be without `backupcopy` set to `yes`.
We can then create a group variable for all docker hosts which enables the `backupcopy` on them.